doc/findinfiles.rst

```diff
@@ -38,21 +39,25 @@ You can select from a number of options to enable searches as broad or refined a
 
 To enable case sensitivity, which only returns matches with identical capitalization to your search text, toggle the :guilabel:`Aa` button on.
 
-[GIF showing search with case sensitive activated]
+.. image:: images/find_in_files/find-in-files-case.gif
+   :alt: Spyder Find pane showing search with case sensitive activated
 
 To parse your search string as a `regular expression`_, enable the :guilabel:`.*` button.
 
 .. _regular expression: https://docs.python.org/3/library/re.html
 
-[GIF showing regular expression search]
+.. image:: images/find_in_files/find-in-files-regex.gif
+   :alt: Spyder Find pane showing regular expression search
 
 To exclude certain filenames, extensions or directories from your search, click the :guilabel:`Plus` button to display the advanced options for the pane, and then enter them in the :guilabel:`Exclude` text box. 
 
-[GIF showing search excluding several file-types]
+.. image:: images/find_in_files/find-in-files-extensions.gif
```

Comment:
Its not vital, but it would be nice if you drew attention to the .po files present before and absent after with your mouse

Reply:
Redoing it

Reply:
I think this is still not clear enough. What do you think about selecting `spyder.po` so it's highlighted when the gif starts? I mean, to show something like this:

![Selección_022](https://user-images.githubusercontent.com/365293/89061862-1655d600-d32b-11ea-8b89-0f3fb57deb5d.png)

Then it'd be clearer that file disappears after adding more the `*.po` exclusion pattern.

Reply:
Thanks @juanis2112! It looks good to me now!